sql/derived_tables/consortial_view.sql

```diff
@@ -0,0 +1,79 @@
+DROP TABLE IF EXISTS reshare_derived.consortial_view;
+
+CREATE TABLE reshare_derived.consortial_view AS
+SELECT
+    'NORTH' AS cv_requester,
+    prr.prr_date_created AS cv_date_created,
+    prr.prr_last_updated AS cv_last_updated,
+    prr.prr_directory_id_fk AS cv_directory_id_fk,
+    prr.prr_patron_request_fk AS cv_patron_request_fk,
+    prr.prr_state_fk AS cv_state_fk,
+    s.st_code AS cv_code
+FROM
+    reshare_reshare_north_rs.patron_request_rota prr
+    JOIN reshare_reshare_north_rs.status s ON s.st_id = prr.prr_state_fk
+WHERE
+    s.st_code = 'REQ_REQUEST_COMPLETE'
+    OR s.st_code = 'REQ_SHIPPED';
+
+INSERT INTO reshare_derived.consortial_view
```

Comment:
Could "UNION ALL" be used here instead of INSERT?  I think it would be good to avoid a partially defined table in the case of an error.

Reply:
Sounds good to me, updated the PR.